### PR TITLE
Feat/new style

### PR DESCRIPTION
**Pull request by Skuth**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -87,12 +155,13 @@ button:hover {
 }
 
 .results-container {
-    margin-top: 20px;
+    max-width: 500px;
```
Comment (cryck) [inline index.css:158]: This does not seem to be enough space for a 5x sticker craft on a 1440p resolution.

File: index.css [hunk 2]
```diff
@@ -241,6 +318,10 @@ footer p {
     }
 }
 
-#infoContainer {
```
Comment (cryck) [inline index.css:244]: This should not be removed, possibly happened due to the merge of master.

**Review by cryck — CHANGES_REQUESTED** [review]
Appreciate your contribution @Skuth! You design looks great. 

I'll probably need to commit something to a different branch to keep the newly added infoContainer to be able to merge this. It will have to wait until tomorrow unfortunately.

File: index.css (hunk 2 above)
Comment (Skuth) [reply index.css:244]: I update the function on script, maybe it's working fine now, when I did pull the new changes from main, happen some conflicts, so I did some changes to be align with the main branch

Comment (cryck) [reply index.css:244]: Awesome, thank you for adjusting the PR. 
Much appreciated!

**Pull request merged**